media-sound/ncmpc/ncmpc-0.52.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="Ncurses client for the Music Player Daemon (MPD)"
+HOMEPAGE="https://www.musicpd.org/clients/ncmpc/ https://github.com/MusicPlayerDaemon/ncmpc"
+SRC_URI="https://www.musicpd.org/download/${PN}/${PV%.*}/${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="async-connect chat-screen +colors doc +help-screen
+key-screen +library-screen lirc lyrics-screen +mouse
+nls outputs-screen pcre search-screen +song-screen test
+"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	virtual/pkgconfig
+	doc? ( dev-python/sphinx )
+"
+DEPEND="
+	dev-libs/libfmt:=
+	media-libs/libmpdclient:=
+	sys-libs/ncurses:=[unicode(+)]
+	lirc? ( app-misc/lirc )
+	pcre? ( dev-libs/libpcre2 )
+"
+RDEPEND="
+	${DEPEND}
+	dev-lang/ruby
```

Comment:
I don't see why you've added an unconditional dep on Ruby. It should use the ruby eclasses, but also, shouldn't it be conditional on USE=lyrics-screen, as the original bug report says too?

Reply:
My bad, I didn't check the plugin. It was removed from the upstream long ago. So the ruby dependency isn't needed at all.